### PR TITLE
Detector templates

### PR DESCRIPTION
**Pull request by bnmajor**
Relies on HEXRD/hexrd#594

### CODE REVIEW

**Review by psavery — APPROVED** [review]
Looks great, Brianna! I pushed a few small changes. Feel free to look over them.

It's ready to merge when you are!

**Comment by bnmajor** [comment]
Looks good to me! As soon as CI passes I'll merge this in :+1:

**Pull request merged**
